homeassistant/components/backup/websocket.py

```diff
@@ -0,0 +1,69 @@
+"""Websocket commands for the Backup integration."""
+import voluptuous as vol
+
+from homeassistant.components import websocket_api
+from homeassistant.core import HomeAssistant, callback
+
+from .const import DOMAIN
+from .manager import BackupManager
+
+
+@callback
+def async_register_websocket_handlers(hass: HomeAssistant) -> None:
+    """Register websocket commands."""
+    websocket_api.async_register_command(hass, handle_info)
+    websocket_api.async_register_command(hass, handle_create)
+    websocket_api.async_register_command(hass, handle_remove)
+
+
+@websocket_api.require_admin
+@websocket_api.websocket_command({vol.Required("type"): "backup/info"})
+@websocket_api.async_response
+async def handle_info(
+    hass: HomeAssistant,
+    connection: websocket_api.ActiveConnection,
+    msg: dict,
+):
```

Comment:
Missing return type annotation.
```py
) -> None:
```

Reply:
These are properly typed with the decorators used.

Reply:
https://github.com/home-assistant/core/pull/67427